packages/react-core/src/demos/examples/Card/CardStatus.tsx

```diff
@@ -140,7 +140,7 @@ export const CardStatus: React.FunctionComponent = () => {
         <GridItem>
           <Flex spaceItems={{ default: 'spaceItemsSm' }}>
             <FlexItem>
-              <CheckCircleIcon color={global_success_color_100.var} />
+              <CheckCircleIcon color={global_color_status_success_default.var} />
```

Comment:
any reason we wouldn't use `<Icon/>` for the icons in this file, too?

Reply:
Nope I don't think so, I will update them here too. Any time I think we don't have to directly use tokens we probably should.